rs/nns/integration_tests/BUILD.bazel

```diff
@@ -417,3 +419,24 @@ rust_ic_test(
     ],
     deps = DEPENDENCIES + DEV_DEPENDENCIES,
 )
+
+rust_ic_test(
+    name = "hotkeys_migration_test",
```

Comment:
This will be removed after deployment?

Reply:
This change has been moved to [this PR](https://github.com/dfinity/ic/pull/1171), but yes that is the current plan

Reply:
Yes, I think we're going to remove it (or maybe convert to a test that still makes sense after the migration, if there's value to be extracted from it for free; let's discuss).